README.md

```diff
@@ -20,7 +20,7 @@ Condition Reference
 Each condition is an expression which may contain different functions. Function may be combined with `&&` or `||` constraints and you can turn back conditions with `!` as NOT.
 
 ### articleExists(column: `string` [, includeUnpublished: `bool`])
-Test if an article exists in the column.
+Test if an article exists in the specific column or section.
```

Comment:
Thanks @zonky2 for improving the descriptions. The functions however describes there self. It's currently used to render the backend help. I also want to automate the README.md description. So it would be really helpful, if you also change the function itself:

https://github.com/contao-community-alliance/merger2/blob/62068a90011ff6a4c595a973136de35e7fa1724a/src/Functions/ArticleExistsFunction.php#L90-L103

The german translation file:
https://github.com/contao-community-alliance/merger2/blob/62068a90011ff6a4c595a973136de35e7fa1724a/src/Resources/translations/merger2.de.xliff#L5-L8

Reply:
fixed https://github.com/contao-community-alliance/merger2/pull/29